src/main/java/mate/academy/util/HashUtil.java

```diff
@@ -7,9 +7,6 @@
 public class HashUtil {
     private static final String CRYPTO_ALGORITHM = "SHA-512";
 
-    public HashUtil() {
-    }
-
```

Comment:
Actually, you need a constructor here, but it must be private, to avoid creating an instance of this class. (If I'm not mistaken, it was in some checklist)